include/navigation/image_database.h

```diff
@@ -102,18 +102,18 @@ class ImageDatabase
     class ImageFileWriter
       : public FrameWriter {
     public:
-        ImageFileWriter(const ImageDatabase &) {}
-        void setImageFormat(std::string);
+        ImageFileWriter(const ImageDatabase &, std::string imageFormat);
```

Comment:
``const std::string&`` maybe?

Reply:
I move it into ``m_ImageFormat`` so we can potentially avoid a copy (not that it'll matter with the 3-char strings we're using here anyway!).